Formula/uutils-coreutils.rb

```diff
@@ -21,9 +21,11 @@ class UutilsCoreutils < Formula
   depends_on "rust" => :build
   depends_on "sphinx-doc" => :build
 
-  conflicts_with "coreutils", because: "uutils-coreutils and coreutils install the same binaries"
-  conflicts_with "aardvark_shell_utils", because: "both install `realpath` binaries"
-  conflicts_with "truncate", because: "both install `truncate` binaries"
+  on_macos do
+    conflicts_with "coreutils", because: "uutils-coreutils and coreutils install the same binaries"
+    conflicts_with "aardvark_shell_utils", because: "both install `realpath` binaries"
+    conflicts_with "truncate", because: "both install `truncate` binaries"
```

Comment:
These seem like conflicts of our own making, given that we already prefix commands provided by this formula with `u`. It seems more natural to me to just stop providing the unprefixed symlinks in `bin` and let users who want them to add the `uubin` to their `PATH` instead. (They can append instead of prepend to prioritise macOS-provided commands whenever applicable.)